lib/server/connect.js

```diff
@@ -105,4 +214,13 @@ function updateListenersCount() {
     var timelineCount = SubHandlers.timeline.length;
     sub.changed('kdInfo', 'listeners-count', {count: timelineCount});
   });
+}
+
+function isAuthorizedSession(type, kadiraInfo) {
```

Comment:
Prefix this as `KadiraDebug._isAuthorizedSession`.

Then we can write unit tests for this.